Finmer.Editor/FormDocumentScene.cs

```diff
@@ -510,6 +510,49 @@ private void trvNodes_DragDrop(object sender, DragEventArgs e)
             Dirty = true;
         }
 
+        public TreeNode GetTreeNodeByKey(string key) 
```

Comment:
I think this function should be private. Why would a random outsider - like, say, the main editor window - need to be able to ask for a particular `TreeNode`?